.github/workflows/ci.yml

```diff
@@ -112,6 +112,9 @@ jobs:
             }, {
               name: '@web/test-runner',
               path: './d2l-test-report-web-test-runner.json'
+            }, {
+              name: 'webdriverio',
+              path: './d2l-test-report-webdriverIO.json'
```

Comment:
Same naming suggestion here, any filenames should do `webdriver-io` instead of `webdriveIO`